pkg/api/spec.go

```diff
@@ -1,5 +1,12 @@
 package api
 
+import (
+	"regexp"
+	"strings"
+)
+
+var releaseNameRegex = regexp.MustCompile("[^a-zA-Z0-9\\-]")
```

Comment:
Its not clear from the name of this what its for -- it looks like it matches characters that are disallowed in a release name?

Reply:
I see that this is copy pasted, so I'm :shrug: on it.